CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Use fchmod to change file permission instead of on open to prevent race conditions [854](https://github.com/greenbone/openvas-scanner/pull/854)
 - Several minor potential security risks in different files, spotted by Code QL [854](https://github.com/greenbone/openvas-scanner/pull/854)
 - Fix plugins upload. Backport #878 [#880](https://github.com/greenbone/openvas/pull/880)
+- Fix Error Message when NVTI chache init failed. Backport #885 [#887](https://github.com/greenbone/openvas/pull/887)
```

Comment:
DIfferent PR numbers

Reply:
What do you mean? Its backport of #885. And this PR is 887

Reply:
Ah ok was my bad